flang/lib/Semantics/check-omp-structure.cpp

```diff
@@ -1656,6 +1664,11 @@ void OmpStructureChecker::ChecksOnOrderedAsStandalone() {
           "Only SINK or SOURCE dependence types are allowed when ORDERED construct is a standalone construct with no ORDERED region"_err_en_US);
     }
   }
+  auto doaClauses = FindClauses(llvm::omp::Clause::OMPC_doacross);
```

Comment:
nit: elsewhere in this file braced initialization is used
```suggestion
  auto doaClauses{FindClauses(llvm::omp::Clause::OMPC_doacross)};
```

See also other cases in this diff